[BUG] Leading slashes in `v2/image` `"file_url"`

## Bug description

While working on #881, after installing the Django Debug Toolbar, I noticed that one of our tests was failing. The URL being tested was `/v2/conditions`. The full stack trace can be viewed below.

Investigating the `/v2/conditions` endpoint and was greated with an error page that read:

```
SuspiciousFileOperation at /v2/images/

The joined path (/img/object_icons/elderberry-inn-icons/conditions/blinded.svg) is located
outside of the base path component (/Users/calum/Documents/code/open5e-api/static)
```

Which pointed me toward the `/v2/images` endpoint where I experienced the same error.

Digging into this error I believe that it is being caused by the leading slash in the `"file_path"` in our V2 Image data. This is used to specify an absolute path rather, which I believe is causing the Django development server to throw a `SuspiciousFileOperation` error when Django tries to reach 'up' belong to repository's root director to the computer's root. **The fix here is to remove the leading '/' on the `"file_url"` fields**.

**The error does not appear if DEBUG mode is off**. So this shouldn't cause any issues on the live API, only during local testing.

I can only assume that this error has emerged because the DDT is somehow interferring with how the other middleware (WhiteNoise?) is handling our static files, specifically on sanitising those leading slashes.

So while there is a deeper issue to unpack here about dev/prod parity, the leading slash in the `"file_path"` field of our image data is nonetheless a data error and ought to be fixed.


## Code 

<details>
<summary>Pytest log</summary>

```
___________________________________________________________________ TestObjects.test_condition_example ___________________________________________________________________

self = <Response [400]>, kwargs = {}

    def json(self, **kwargs):
        r"""Decodes the JSON response body (if any) as a Python object.
    
        This may return a dictionary, list, etc. depending on what is in the response.
    
        :param \*\*kwargs: Optional arguments that ``json.loads`` takes.
        :raises requests.exceptions.JSONDecodeError: If the response body does not
            contain valid json.
        """
    
        if not self.encoding and self.content and len(self.content) > 3:
            # No encoding set. JSON RFC 4627 section 3 states we should expect
            # UTF-8, -16 or -32. Detect which one to use; If the detection or
            # decoding fails, fall back to `self.text` (using charset_normalizer to make
            # a best guess).
            encoding = guess_json_utf(self.content)
            if encoding is not None:
                try:
                    return complexjson.loads(self.content.decode(encoding), **kwargs)
                except UnicodeDecodeError:
                    # Wrong UTF codec detected; usually because it's not UTF-8
                    # but some other 8-bit codec.  This is an RFC violation,
                    # and the server didn't bother to tell us what codec *was*
                    # used.
                    pass
                except JSONDecodeError as e:
                    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
    
        try:
>           return complexjson.loads(self.text, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

../../../.local/share/virtualenvs/open5e-api-zZJBumHW/lib/python3.11/site-packages/requests/models.py:976: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/__init__.py:346: in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
/usr/local/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/decoder.py:337: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <json.decoder.JSONDecoder object at 0x10cac0450>
s = 'SuspiciousFileOperation at /v2/conditions/stunned/\nThe joined path (/img/object_icons/elderberry-inn-icons/condition...ile. Change that to False, and Django will\ndisplay a standard page generated by the handler for this status code.\n\n'
idx = 0

    def raw_decode(self, s, idx=0):
        """Decode a JSON document from ``s`` (a ``str`` beginning with
        a JSON document) and return a 2-tuple of the Python
        representation and the index in ``s`` where the document ended.
    
        This can be used to decode a JSON document from a string that may
        have extraneous data at the end.
    
        """
        try:
            obj, end = self.scan_once(s, idx)
        except StopIteration as err:
>           raise JSONDecodeError("Expecting value", s, err.value) from None
E           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

/usr/local/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/decoder.py:355: JSONDecodeError

During handling of the above exception, another exception occurred:

self = <test_objects.TestObjects object at 0x1295c7750>

    def test_condition_example(self):
        path="/v2/conditions/stunned/"
>       self._verify(path)

api_v2/tests/test_objects.py:171: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
api_v2/tests/test_objects.py:49: in _verify
    response = requests.get(API_BASE + endpoint, allow_redirects=True, headers = {'Accept': 'application/json'}).json()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <Response [400]>, kwargs = {}

    def json(self, **kwargs):
        r"""Decodes the JSON response body (if any) as a Python object.
    
        This may return a dictionary, list, etc. depending on what is in the response.
    
        :param \*\*kwargs: Optional arguments that ``json.loads`` takes.
        :raises requests.exceptions.JSONDecodeError: If the response body does not
            contain valid json.
        """
    
        if not self.encoding and self.content and len(self.content) > 3:
            # No encoding set. JSON RFC 4627 section 3 states we should expect
            # UTF-8, -16 or -32. Detect which one to use; If the detection or
            # decoding fails, fall back to `self.text` (using charset_normalizer to make
            # a best guess).
            encoding = guess_json_utf(self.content)
            if encoding is not None:
                try:
                    return complexjson.loads(self.content.decode(encoding), **kwargs)
                except UnicodeDecodeError:
                    # Wrong UTF codec detected; usually because it's not UTF-8
                    # but some other 8-bit codec.  This is an RFC violation,
                    # and the server didn't bother to tell us what codec *was*
                    # used.
                    pass
                except JSONDecodeError as e:
                    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
    
        try:
            return complexjson.loads(self.text, **kwargs)
        except JSONDecodeError as e:
            # Catch JSON-related errors and raise as requests.JSONDecodeError
            # This aliases json.JSONDecodeError and simplejson.JSONDecodeError
>           raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
E           requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

../../../.local/share/virtualenvs/open5e-api-zZJBumHW/lib/python3.11/site-packages/requests/models.py:980: JSONDecodeError
======================================================================== short test summary info =========================================================================
FAILED api_v2/tests/test_objects.py::TestObjects::test_condition_example - requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
===================================================================== 1 failed, 59 passed in 15.19s ======================================================================
```
</details>

--- a/data/v2/open5e/elderberry-inn-icons/Image.json
+++ b/data/v2/open5e/elderberry-inn-icons/Image.json
@@ -4,7 +4,7 @@
     "alt_text": "An icon representing the blinded condition in an RPG. A simple, abstract black-and-white icon of an eye with a line crossed through it.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/blinded.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/blinded.svg",
     "name": "Blinded",
     "type": "icon"
   },
@@ -16,7 +16,7 @@
     "alt_text": "An icon representing the charmed condition in an RPG. A simple black-and-white silhouette of a humanoid facing the viewer. There is a heart where their face should be.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/charmed.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/charmed.svg",
     "name": "Charmed",
     "type": "icon"
   },
@@ -28,7 +28,7 @@
     "alt_text": "An icon representing the deafened condition in an RPG. A simple black-and-white outline of an ear with a diagonal line passing through it, as if it were crossed out.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/deaftened.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/deaftened.svg",
     "name": "Deaftened",
     "type": "icon"
   },
@@ -40,7 +40,7 @@
     "alt_text": "An icon representing the Exhaustion (one level) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '1'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/exhaustion1.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/exhaustion1.svg",
     "name": "Exhaustion1",
     "type": "icon"
   },
@@ -52,7 +52,7 @@
     "alt_text": "An icon representing the Exhaustion (two levels) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '2'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/exhaustion2.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/exhaustion2.svg",
     "name": "Exhaustion2",
     "type": "icon"
   },
@@ -64,7 +64,7 @@
     "alt_text": "An icon representing the Exhaustion (three levels) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '3'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/exhaustion3.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/exhaustion3.svg",
     "name": "Exhaustion3",
     "type": "icon"
   },
@@ -76,7 +76,7 @@
     "alt_text": "An icon representing the Exhaustion (four levels) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '4'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/exhaustion4.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/exhaustion4.svg",
     "name": "Exhaustion4",
     "type": "icon"
   },
@@ -88,7 +88,7 @@
     "alt_text": "An icon representing the Exhaustion (five levels) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '5'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/exhaustion5.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/exhaustion5.svg",
     "name": "Exhaustion5",
     "type": "icon"
   },
@@ -100,7 +100,7 @@
     "alt_text": "An icon representing the Exhaustion (six levels) condition. A simple black-and-white diagram of a standing figure hunched over in exhaustion with a large numeral '6'.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/exhaustion6.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/exhaustion6.svg",
     "name": "Exhaustion6",
     "type": "icon"
   },
@@ -112,7 +112,7 @@
     "alt_text": "An icon representing the Frightened condition. A simple black-and-white silhouette figure from the shoulders up facing the viewer. Their hands are raised to their head and they are screaming.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/frightened.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/frightened.svg",
     "name": "Frightened",
     "type": "icon"
   },
@@ -124,7 +124,7 @@
     "alt_text": "An icon representing the Grappled condition. A simple black-and-white image of an arm staining against a chain and shackle attached to the wrist.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/grappled.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/grappled.svg",
     "name": "Grappled",
     "type": "icon"
   },
@@ -136,7 +136,7 @@
     "alt_text": "An icon representing the Incapacitated condition. A black-and-white silhouetted figure visible from the shoulders up faces the viewer. There is an X where thier face should be, and a shirling figure over thier head.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/incapacitated.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/incapacitated.svg",
     "name": "Incapacitated",
     "type": "icon"
   },
@@ -148,7 +148,7 @@
     "alt_text": "An icon representing the Inconscious condition. A black-and-white silhouetted figure visible from the shoulders up faces the viewer. Where its eyes should be are swirls",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/inconscious.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/inconscious.svg",
     "name": "Inconscious",
     "type": "icon"
   },
@@ -160,7 +160,7 @@
     "alt_text": "An icon representing the Invisible condition. A black-and-white silhouette of a figure visible from the shoulders up faces the viewer. They are drawn with a dashed outline to indicate that they are invisible.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/invisible.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/invisible.svg",
     "name": "Invisible",
     "type": "icon"
   },
@@ -172,7 +172,7 @@
     "alt_text": "An icon representing the Paralyzed condition. A black-and-white silhouette of a figure visible from the shoulders up faces the viewer. They are surrounded by a chain.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/paralyzed.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/paralyzed.svg",
     "name": "Paralyzed",
     "type": "icon"
   },
@@ -184,7 +184,7 @@
     "alt_text": "An icon representing the Petrified condition. A black-and-white silhouette of a figure visible from the shoulders up faces the viewer. They are covered in numerous broad cracks as if they were an old, crumbling statue.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/petrified.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/petrified.svg",
     "name": "Petrified",
     "type": "icon"
   },
@@ -196,7 +196,7 @@
     "alt_text": "An icon representing the Poisoned condition. A simple black-and-white drawing of a bottle with a stopper and a skull on its front.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/poisoned.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/poisoned.svg",
     "name": "Poisoned",
     "type": "icon"
   },
@@ -208,7 +208,7 @@
     "alt_text": "An icon representing the Prone condition.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/prone.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/prone.svg",
     "name": "Prone",
     "type": "icon"
   },
@@ -220,7 +220,7 @@
     "alt_text": "An icon representing the Restrained condition. A simple black-and-white silouetted figure is on all fours, as if crawling. Three large arrows above the figure point downward.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/restrained.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/restrained.svg",
     "name": "Restrained",
     "type": "icon"
   },
@@ -232,7 +232,7 @@
     "alt_text": "An icon representing the Stunned condition. A black-and-white silhouette of a figure visible from the shoulders up faces the viewer. They are surrounded by stars that sit on rings which suggest orbits.",
     "attribution": "Designed with love by Anaislalovi (@anaislalovi) for Elderberry Inn.",
     "document": "elderberry-inn-icons",
-    "file_path": "/img/object_icons/elderberry-inn-icons/conditions/stunned.svg",
+    "file_path": "img/object_icons/elderberry-inn-icons/conditions/stunned.svg",
     "name": "Stunned",
     "type": "icon"
   },

--- a/data/v2/open5e/open5e/Image.json
+++ b/data/v2/open5e/open5e/Image.json
@@ -4,7 +4,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/aboleth.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/aboleth.png",
     "name": "Aboleth",
     "type": "illustration"
   },
@@ -16,7 +16,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/ankheg.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/ankheg.png",
     "name": "Ankheg",
     "type": "illustration"
   },
@@ -28,7 +28,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/basilisk.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/basilisk.png",
     "name": "Basilisk",
     "type": "illustration"
   },
@@ -40,7 +40,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/behir.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/behir.png",
     "name": "Behir",
     "type": "illustration"
   },
@@ -52,7 +52,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/bulette.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/bulette.png",
     "name": "Bulette",
     "type": "illustration"
   },
@@ -64,7 +64,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/chuul.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/chuul.png",
     "name": "Chuul",
     "type": "illustration"
   },
@@ -76,7 +76,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/cloaker.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/cloaker.png",
     "name": "Cloaker",
     "type": "illustration"
   },
@@ -88,7 +88,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/cockatrice.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/cockatrice.png",
     "name": "Cockatrice",
     "type": "illustration"
   },
@@ -100,7 +100,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/couatl.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/couatl.png",
     "name": "Couatl",
     "type": "illustration"
   },
@@ -112,7 +112,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/darkmantle.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/darkmantle.png",
     "name": "Darkmantle",
     "type": "illustration"
   },
@@ -124,7 +124,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/hezrou.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/hezrou.png",
     "name": "Hezrou",
     "type": "illustration"
   },
@@ -136,7 +136,7 @@
     "alt_text": "",
     "attribution": "",
     "document": "open5e",
-    "file_path": "/img/object_illustrations/open5e-illustrations/monsters/rust-monster.png",
+    "file_path": "img/object_illustrations/open5e-illustrations/monsters/rust-monster.png",
     "name": "Rust Monster",
     "type": "illustration"
   },

